test/unit/rbtree_test.js

```diff
@@ -20,10 +20,11 @@
 
 var RBTree = require('../../lib/ring/rbtree').RBTree;
 var RingNode = require('../../lib/ring/rbtree').RingNode;
+var comparator = require('../../lib/ring').comparator;
```

Comment:
The next PR does exactly this. For this PR, I prefer to re-use the comparator from `ring/index.js` instead of duplicating the same function in the tests